toplevel/coqloop.ml

```diff
@@ -438,19 +438,15 @@ let rec loop ~state =
       loop ~state
 
 (* Default toplevel loop *)
-let warning s = Flags.(with_option warn Feedback.msg_warning (strbrk s))
 
 let drop_args = ref None
+
 let loop ~opts ~state =
   drop_args := Some opts;
   let open Coqargs in
   print_emacs := opts.config.print_emacs;
   (* We initialize the console only if we run the toploop_run *)
   let tl_feed = Feedback.add_feeder coqloop_feed in
-  if Dumpglob.dump () then begin
-    Flags.if_verbose warning "Dumpglob cannot be used in interactive mode.";
-    Dumpglob.noglob ()
-  end;
```

Comment:
Why to suppress this? This means that `coqtop -dump-glob foo.glob` will silently do nothing when a dump option is given. Either the warning should be kept (with instead a test on `opts.config.glob_out`) or, better, since the glob file has no purpose in an interactive session with ephemere locations, the options `-dump-glob`, `-no-glob` and `feedback-glob` should be moved from `coqargs` to `coqcargs`.

Reply:
I have moved the option to the `coqc` arguments as you suggested, thanks.